src/backends/perf_pt/collect.c

```diff
@@ -292,14 +292,16 @@ read_aux(void *aux_buf, struct perf_event_mmap_page *hdr,
         trace->buf = new_buf;
     }
 
-    // Finally copy the AUX buffer into the trace buffer.
+    // Finally append a copy the new AUX data onto the trace storage buffer.
```

Comment:
Slightly awkward phrasing with "onto".

Reply:
"append a copy the new AUX data to the trace storage buffer."?

Reply:
How about just "Add the AUX data to the end of the trace buffer" or similar?